composer.json

```
@@ -86,7 +89,7 @@
         "tubalmartin/cssmin": "^4.1",
         "web-token/jwt-framework": "^3.1",
         "webonyx/graphql-php": "^14.11",
-        "wikimedia/less.php": "^3.0"
+        "wikimedia/less.php": "^4.0"
```

What's not compatible with version `3.0` in this pull request?
```suggestion
        "wikimedia/less.php": "^3.0 || ^4.0"
```

---

seem up to v4 bring more fixes for packages. I thinks we safe to use v4 with mininum support php v 7.4+

---

https://gerrit.wikimedia.org/g/mediawiki/libs/less.php/+log/v4.0.0

---

Hey,

it was upgraded when magento 2.4.6 was not released and when I `setup:static-content:deploy`, it returns error (apology I hadn't take a screenshot of it).

once I get latest magento 2.4.6, it is working, so will revert it back to ^3.0

thanks for the review

---

"wikimedia/less.php": "^3.2"
current we using this ver. If you plan to upgrade to v4 you should consider re-check less compilation by run setup static content deploy

---

I already tried, I tried deploy using version ^3.2 and version ^4.0, everything  was fine. is it fine to upgrade `wikimedia/less.php:^4.0` on `lib/internal/Magento/Framework/composer.json`?

I reverted to 3.2 as Magento/Framework is requiring version of `^3.2`.